nsxt/data_source_nsxt_policy_ipsec_vpn_local_endpoint.go

```diff
@@ -0,0 +1,117 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/tier_0s/locale_services/ipsec_vpn_services"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func dataSourceNsxtPolicyIPSecVpnLocalEndpoint() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNsxtPolicyIPSecVpnLocalEndpointRead,
+
+		Schema: map[string]*schema.Schema{
+			"id":           getDataSourceIDSchema(),
+			"display_name": getDataSourceDisplayNameSchema(),
+			"description":  getDataSourceDescriptionSchema(),
+			"path":         getPathSchema(),
+			"tier0_id": {
+				Type:        schema.TypeString,
+				Description: "Policy path referencing Local endpoint.",
+				Optional:    true,
+				Default:     "vmc",
+			},
+			"locale_service": {
+				Type:        schema.TypeString,
+				Description: "Local_service",
+				Optional:    true,
+				Default:     "default",
+			},
+			"service_id": {
```

Comment:
Is this path of endpoint as indicated in description? In this case `endpoint_path` seems like a better fit